src/components/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+
+type Props = {
+  name: string;
+  clickHandler(): void;
+};
+
+export const Button: FC<Props> = ({ name, clickHandler }) => (
+  <button
+    type="button"
+    onClick={() => {
+      clickHandler();
+    }}
```

Comment:
you can add just callback if you don't pass any variables inside this func

```suggestion
    onClick={clickHandler}
```